ntbs-service/wwwroot/source/Components/ValidateDate.ts

```diff
@@ -36,11 +35,26 @@ const ValidateDate = Vue.extend({
                     console.log(response);
                     var errorMessage = response.data;
                     this.hasError = errorMessage != '';

```

Comment:
So are we not using this.hasError in the views anymore, and instead just directly set the classes in the vue components? Shouldn't hasError be a local variable then rather than in date, or are we still using it for something else in the views?

Reply:
There was a bit that had hasError binded, needed to fix that. now this file doesn't need binded hasError